comictaggerlib/renamewindow.py

```diff
@@ -82,7 +84,7 @@ def config_renamer(self, ca: ComicArchive, md: GenericMetadata | None = None) ->
             new_ext = ca.extension()
 
         if md is None or md.is_empty:
-            md = ca.read_metadata(self.data_style)
+            md, success = self.parent().overlay_ca_read_style(self.load_data_styles, ca)
```

Comment:
`success` is never used, either remove the return value from `overlay_ca_read_style` or do something with it here